src/hotspot/share/gc/parallel/psGCAdaptivePolicyCounters.hpp

```diff
@@ -1,185 +0,0 @@
-    _old_eden_size->set_value(old_size);
-  }
-  inline void update_old_promo_size(size_t old_size) {
-    _old_promo_size->set_value(old_size);
-  }
-  inline void update_avg_promoted_avg() {
-    _avg_promoted_avg_counter->set_value(
-      (jlong)(ps_size_policy()->avg_promoted()->average())
-    );
-  }
-  inline void update_avg_promoted_dev() {
-    _avg_promoted_dev_counter->set_value(
-      (jlong)(ps_size_policy()->avg_promoted()->deviation())
-    );
-  }
-  inline void update_avg_promoted_padded_avg() {
-    _avg_promoted_padded_avg_counter->set_value(
-      (jlong)(ps_size_policy()->avg_promoted()->padded_average())
-    );
-  }
-
-  inline void update_avg_pretenured_padded_avg() {
-    _avg_pretenured_padded_avg->set_value(
-      (jlong)(ps_size_policy()->_avg_pretenured->padded_average())
-    );
-  }
-  inline void update_change_young_gen_for_maj_pauses() {
-    _change_young_gen_for_maj_pauses_counter->set_value(
-      ps_size_policy()->change_young_gen_for_maj_pauses());
-  }
-  inline void update_change_old_gen_for_min_pauses() {
-    _change_old_gen_for_min_pauses->set_value(
-      ps_size_policy()->change_old_gen_for_min_pauses());
-  }
-
-  // compute_generations_free_space() statistics
-
-  inline void update_avg_major_pause() {
-    _avg_major_pause->set_value(
-      (jlong)(ps_size_policy()->_avg_major_pause->average() * 1000.0)
-    );
-  }
-  inline void update_avg_major_interval() {
-    _avg_major_interval->set_value(
-      (jlong)(ps_size_policy()->_avg_major_interval->average() * 1000.0)
-    );
-  }
-
-  inline void update_major_gc_cost_counter() {
-    _major_gc_cost_counter->set_value(
-      (jlong)(ps_size_policy()->major_gc_cost() * 100.0)
-    );
-  }
-  inline void update_mutator_cost_counter() {
-    _mutator_cost_counter->set_value(
-      (jlong)(ps_size_policy()->mutator_cost() * 100.0)
-    );
-  }
-
-  inline void update_live_space() {
-    _live_space->set_value(ps_size_policy()->live_space());
-  }
-  inline void update_free_space() {
-    _free_space->set_value(ps_size_policy()->free_space());
-  }
-
-  inline void update_avg_old_live() {
-    _avg_old_live_counter->set_value(
-      (jlong)(ps_size_policy()->avg_old_live()->average())
-    );
-  }
-  // Scale up all the slopes
-  inline void update_major_pause_old_slope() {
-    _major_pause_old_slope->set_value(
-      (jlong)(ps_size_policy()->major_pause_old_slope() * 1000)
-    );
-  }
-  inline void update_minor_pause_old_slope() {
-    _minor_pause_old_slope->set_value(
-      (jlong)(ps_size_policy()->minor_pause_old_slope() * 1000)
-    );
-  }
-  inline void update_major_pause_young_slope() {
-    _major_pause_young_slope->set_value(
-      (jlong)(ps_size_policy()->major_pause_young_slope() * 1000)
-    );
-  }
-  inline void update_gc_overhead_limit_exceeded_counter() {
-    gc_overhead_limit_exceeded_counter()->set_value(
-      (jlong) ps_size_policy()->gc_overhead_limit_exceeded());
-  }
```

Comment:
The field `GCPolicyCounters::_gc_overhead_limit_exceeded_counter` and the related code are not used now. It is good to remove them.